akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/internal/BehaviorTestKitImpl.scala

```diff
@@ -200,9 +200,11 @@ private[akka] object BehaviorTestKitImpl {
 
     def inteceptBehaviour[T](behavior: Behavior[T], ctx: TypedActorContext[T]): Behavior[T] =
       Behavior
-        .start(Behaviors.intercept { () =>
-          this.asInstanceOf[BehaviorInterceptor[Any, T]]
-        }(behavior), ctx.asInstanceOf[TypedActorContext[Any]])
+        .start(
+          Behaviors.intercept { () =>
+            this.asInstanceOf[BehaviorInterceptor[Any, T]]
+          }(behavior),
+          ctx.asInstanceOf[TypedActorContext[Any]])
```

Comment:
I think the old one is better.

Reply:
I think this can be improved the various `newlines.*` `fold`/`unfold` options however doing this ends up creating a ginormous diff because while it improves some sections in the codebase (such as this one), a large portion of other code ends up getting updated.

You can play around around with this locally if you want, generally speaking the area you want to look at is `newlines` under https://scalameta.org/scalafmt/docs/configuration.html#newlines. If you can find an option that reduces the diff more while keeping the niceness that would be great!

Reply:
Agree, the old is better

Reply:
So I played around a bit to try and replicate the original behaviour and didn't get anywhere. Made a comment on scalafmt's gitter at https://gitter.im/scalameta/scalafmt?at=634fd35127f328266d474ea0, I would say that if a solution isn't found lets just accept it.

Reply:
the old formatting was a bug since it ignored the braces block when checking everything fits on one line -- which it doesn't.

if you like it when multiple arguments are on the same line, you could try https://scalameta.org/scalafmt/docs/configuration.html#binpackunsafexxxsite but it will lead to a larger diff, most likely.

Reply:
Thanks! I will play around with it but as you said I suspect that it will result in a larger diff.

Reply:
This actually ended up working and the diff is actually smaller, thanks for the tip.